### DISCUSSION TITLE
Strange timeout issues when running ava thorugh VSCode

### DISCUSSION THREAD

**Discussion by simplenotezy** [avajs/ava#3083] [Q&A]
I am trying to run ava through VS Code in "Run and debug", so I can step through my tests.

But it is stuck out for strange issues and without providing any errors. It's simply stuck at `Debugger attached.`.

Sometimes the VS Code debug tool will put me inside `clean-yaml-object` node_modules library which is depended upon by AVA.

I wonder if my configuration is correct? It's worth noting that it used to work fine, but not anymore.

```json
    {
      "type": "node",
      "request": "launch",
      "name": "Debug AVA test file",
      "runtimeExecutable": "${workspaceFolder}/node_modules/.bin/ava",
      "runtimeArgs": ["${file}", "--verbose", "--no-worker-threads", "--fail-fast"],
      "console": "integratedTerminal",
      "internalConsoleOptions": "neverOpen",
      "outputCapture": "std",
      "disableOptimisticBPs": true,
      "restart": true,
      "skipFiles": ["<node_internals>/**/*.js"]
    },
```

It currently runs fine if I run `npm test`, however, during today I've had similar issues in the terminal

**Comment by novemberborn** [comment]
The recommended configuration has changed a little, perhaps adjusting it to match https://github.com/avajs/ava/blob/main/docs/recipes/debugging-with-vscode.md will help?
